modules/references/tests/server/reference.server.jobs.tests.js

```diff
@@ -44,21 +44,23 @@ describe('Job: Set reference to public after a given period of time', function (
     return ['yes', 'no', 'unknown'][Math.floor(3 * Math.random())];
   }
 
-  var referenceData = {
-    get userFrom() { return generateRandomId(); },
-    get userTo() { return generateRandomId(); },
-    // here we don't care if all interactions are false
-    get met() { return generateRandomBoolean(); },
-    get hostedMe() { return generateRandomBoolean(); },
-    get hostedThem() { return generateRandomBoolean(); },
-    get recommend() { return generateRandomRecommend(); }
-  };
+  function generateReferenceData() {
+    return {
+      userFrom: generateRandomId(),
+      userTo: generateRandomId(),
+      // here we don't care if all interactions are false
+      met: generateRandomBoolean(),
+      hostedMe: generateRandomBoolean(),
+      hostedThem: generateRandomBoolean(),
+      recommend: generateRandomRecommend()
```

Comment:
If these are random, they could probably simply be `false` instead since we're not testing for their values anyway.

Reply:
resolved with ba25d1c